models/deepseek_v4_flash_mtp/prefill_cp_hca.py

```
@@ -0,0 +1,2140 @@
+                tensors["overlay_active_lengths"][owner, part, tile, 1]
+            )
+            leaves.append(
+                (
+                    local_norm[owner, part, tile],
+                    tensors["query_positions"][owner, part, tile],
+                    active,
+                    True,
+                )
+            )
+
+        for leaf, (leaf_x, positions, active, publish) in enumerate(leaves):
```

Ruff (0.16.1)

[warning] Loop control variable `leaf` not used within loop body

Rename unused `leaf` to `_leaf`

(B007)